docs/root/operations/cli.rst

```diff
@@ -67,6 +67,12 @@ following are the command line options that Envoy supports.
   :option:`--restart-epoch` is non-zero. Instead, for subsequent hot restarts, set
   :option:`--base-id` option with the selected base ID. See :option:`--base-id-path`.
 
+.. option:: --skip-hot-restart-on-no-parent
```

Comment:
is it worth a release note?

Reply:
Added a release note. I don't know if it's worth it, I think using hot restart is extremely niche, but no harm!